README.md

```diff
@@ -2,7 +2,11 @@
 Embedding a video in a webpage
 
 BUILT WITH
+-HTML
 
-HTML
+-CSS
 
-CSS
+
+Images
+![GitHub Logo](/images/embedding-video.png)
+Format: ![Alt Text](/images/embedding-video.png)
```

Comment:
Let's make this more descriptive. See below:
```md
## Screenshot
![App Screenshot](/images/embedding-video.png)
```

Please delete `Format: ![Alt Text](/images/embedding-video.png)`  as it is redundant here. Also, let's make sure the file been used also have the name `embedding-video.png`